src/main/java/com/epam/tc/Operations.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.tc;
+
+import com.epam.tat.module4.Calculator;
+
+public class Operations {

```

Comment:
Why do you need this class?

Reply:
Here I created own methods using calculator.jar methods. And use this operation methods for testing addiction, Subtraction,multiply and division.